bin/compile

```diff
@@ -91,13 +96,18 @@ install_bins() {
     warn_node_engine "$iojs_engine"
     install_iojs "$iojs_engine" "$BUILD_DIR/.heroku/node"
     echo "Using bundled npm version for iojs compatibility: `npm --version`"
+    mcount 'bin.iojs.$(node --version)'
```

Comment:
consider `version.iojs.$(version)`. I think I would more readily understand that downstream then `bin.iojs.${version}`. P

You will get a proliferation of metrics, one for every conceivable version of node, yard, npm in existence. 

You are also taking raw output from a program so you are susceptible to silently loosing metrics later on if say `node --version` changes from `v7.6.0` to `node version 7.6.0`.